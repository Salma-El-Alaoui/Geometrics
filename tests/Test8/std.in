DELETE c1
C c1 1 1 1
LIST
DELETE c2
LIST
R r1 1 1 1 1
L l1 1 1 1 1
LIST
DELETE c1
DELETE c1
LIST
DELETE l1
DELETE
LIST
EXIT
