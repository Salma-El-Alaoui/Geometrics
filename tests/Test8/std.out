ERR
#L'element geometrique nomme c1 n'existe pas.
OK
#Element ajoute: c1
C c1 1 1 1
ERR
#L'element geometrique nomme c2 n'existe pas.
C c1 1 1 1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
C c1 1 1 1
L l1 1 1 1 1
R r1 1 1 1 1
#Element c1 supprime
OK
ERR
#L'element geometrique nomme c1 n'existe pas.
L l1 1 1 1 1
R r1 1 1 1 1
#Element l1 supprime
OK
ERR
#Parametres invalides
R r1 1 1 1 1
